core/src/main/java/com/yahoo/imapnio/async/data/ExtensionMailboxInfo.java

```diff
@@ -49,8 +58,15 @@ public ExtensionMailboxInfo(@Nonnull final IMAPResponse[] resps) throws ParsingE
                 if (values != null && values.length >= 1) {
                     mailboxId = values[0];
                     resps[i] = null; // Nulls out this element in array to be consistent with MailboxInfo behavior
-                    break;
                 }
+            } else if (ir.isOK() && key.equals(CLOSED_ID)) { // example * OK [CLOSED]
```

Comment:
Probably, we should not do this. From RFC, CLOSED seems to be a positional response. From RFC, any untagged folder responses after CLOSED should be the ones considered for mailbox info.

Reply:
Should i just pass the response similar to OK response and caller can decide?